src/mavlink_stream.c

```diff
@@ -24,6 +25,11 @@ static void mavlink_send_heartbeat(void)
                              MAV_STATE_STANDBY);
 }
 
+static uint32_t time_mavlink_send_message(void* f)
```

Comment:
Sorry, I forgot to delete this.  It is a leftover from one of my attempts and correcting the mavlink send timing.